### PR TITLE
docs: Added Invoice Discounting Documentation

### PR DESCRIPTION
**Pull request by asbasawaraj**

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<!-- add-breadcrumbs -->
+
+**Invoice discounting is the practice of using a company's unpaid invoices as collateral for a short term loan, which is issued by a bank or a finance company.**
+
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]: add a paragraph demonstrating a scenario to add clarity on use case
like: consider that you have some unpaid invoices for xyz and you need money...etc

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+### 1. Prerequisite
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
To access the Invoice Discounting list, go to:
> Home > Accounting > Banking and Payments > Invoice Discounting

Prerequisites
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+You need to create below ledgers in order to post invoice discounting transactions.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
You need to create following ledgers in order to post invoice discounting transactions:
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+**Short Term Loan:** A ledger under current liabilities for loan.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
**Short Term Loan:** A ledger under the 'Current Liabilities' account for loan.
```

Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]: these definitions are too short, need more explanation

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+**Accounts Receivable Credit Account:** A control account of type receivable.
+
+**Accounts Receivable Discounted Account:** A receivable account for invoices which have been discounted.
+
+**Accounts Receivable Unpaid Account:** A receivable account for invoices which were discounted and remain unpaid even after the loan period is over.
+
+
+
+### 2. How to Post an Invoice Discounting Transaction
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
## 1. How to Post an Invoice Discounting Transaction
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+To post a new Invoice Discounting:
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
```
this is the title, no need to repeat

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+1. Go to **Accounting > Banking and Payments > Invoice Discounting > New**.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
1. Go to the Invoice Discounting list, click on New.
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+1. Enter Posting Date, Loan Start Date, Loan Period.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
1. Enter Posting Date and Loan Start Date. Enter the Loan Period in days.
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+1. Select invoices either manually or by clicking on 'Get Invoices'.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
1. Select invoices either manually in the table or by clicking on the 'Get Invoices' button on the top right.
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+1. Select Short Term Loan Account, Bank Account, Bank Charges Account.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
1. Select Short Term Loan Account, Bank Account, and Bank Charges Account.
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+1. Select Accounts Receivable Credit Account, Accounts Receivable Discounted Account and Accounts Receivable Unpaid Account
+1. Click on Save and Submit.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
1. Click on Save then Submit.
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+1. Click on 'Disburse Loan' and in the Journal Entry screen click on Save and Submit.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]: unclear
After submitting, click on 'Disburse Loan'. Go to Journal Entry....?

also, a screenshot for the journal entry screen?

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+
+  <img class="screenshot" alt="Lead" src="{{docs_base_url}}/assets/img/accounts/invoice_discounting.png">
+
+### 3. Features
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
## 2. Features
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+
+#### 3.1 Import Invoices
+You can click on 'Get Invoices' button to import invoices. You can import invoices based on certain criteria.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
Click on 'Get Invoices' button to import invoices. You can import invoices based on certain criteria like:
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+
+1. Invoices raised to specific customer.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]: regular bullets not numbered

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md (hunk 13 above)
Comment (asbasawaraj) [reply erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]: 1 is Prerequisite, 2 is How to Post an... 3 is faetures

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md (hunk 4 above)
Comment (asbasawaraj) [reply erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]: I am not referring to the 'Current Liabilities' group specifically...

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md (hunk 2 above)
Comment (asbasawaraj) [reply erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]: I didn't get you point...

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md (hunk 1 above)
Comment (asbasawaraj) [reply erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]: I have covered that in the definition...

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md (hunk 2 above)
Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]: for mat is 1. How to create, so prerequisites shouldn't be numbered. ref: feedback from umair on another page

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md (hunk 4 above)
Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]: what are you referring to? the sentence is not clear for the end user

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md (hunk 13 above)
Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]: after removing the number 1. from prerequisite

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 16]
```diff
@@ -0,0 +1,50 @@
...
+1. Enter Posting Date and Loan Start Date. Enter the Loan Period in days.
+1. Select invoices either manually in the table or by clicking on the 'Get Invoices' button on the top right.
+1. Select Short Term Loan Account, Bank Account, and Bank Charges Account.
+1. Select Accounts Receivable Credit Account, Accounts Receivable Discounted Account and Accounts Receivable Unpaid Account
+1. Click on Save then Submit.
+1. Click on 'Disburse Loan' and a Journal Entry screen will be prepared. Save then Submit the Journal Entry.
+
+  <img class="screenshot" alt="Lead" src="{{docs_base_url}}/assets/img/accounts/invoice_discounting.png">
+
+### 1. Features
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
### 2. Features
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 17]
```diff
@@ -0,0 +1,50 @@
...
+* Invoices raised to specific customer.
+* Date range between which the invoices were raised.
+* Minimum and maximum amount.
+
+You can also specify multiple of the above filters.
+
+#### 2.2 Close the Loan
+When you repay the loan at the end of the loan period or before that, you can update the same by clicking on 'Close Loan' button. System will prepare the Journal Entry. Review and Submit the same.
+
+#### 2.3 Auto Update of Ledgers at the End of Loan Period
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
### 2.3 Auto Update of Ledgers at the End of Loan Period
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 18]
```diff
@@ -0,0 +1,50 @@
+<!-- add-breadcrumbs -->
+
+**Invoice discounting is the practice of using a company's unpaid invoices as collateral for a short term loan, which is issued by a bank or a finance company.**
+
+### Prerequisite
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
### Prerequisites
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 19]
```diff
@@ -0,0 +1,50 @@
+<!-- add-breadcrumbs -->
+
+**Invoice discounting is the practice of using a company's unpaid invoices as collateral for a short term loan, which is issued by a bank or a finance company.**
+
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
To access the Invoice discounting list, go to:
> Home > Accounting > Banking and Payments > Invoice Discounting
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 20]
```diff
@@ -0,0 +1,50 @@
...
+#### 2.1 Import Invoices
+Click on 'Get Invoices' button to import invoices. You can import invoices based on certain criteria.
+
+* Invoices raised to specific customer.
+* Date range between which the invoices were raised.
+* Minimum and maximum amount.
+
+You can also specify multiple of the above filters.
+
+#### 2.2 Close the Loan
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
### 2.2 Close the Loan
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 21]
```diff
@@ -0,0 +1,50 @@
...
+1. Select Short Term Loan Account, Bank Account, and Bank Charges Account.
+1. Select Accounts Receivable Credit Account, Accounts Receivable Discounted Account and Accounts Receivable Unpaid Account
+1. Click on Save then Submit.
+1. Click on 'Disburse Loan' and a Journal Entry screen will be prepared. Save then Submit the Journal Entry.
+
+  <img class="screenshot" alt="Lead" src="{{docs_base_url}}/assets/img/accounts/invoice_discounting.png">
+
+### 1. Features
+
+#### 2.1 Import Invoices
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
### 2.1 Import Invoices
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 22]
```diff
@@ -0,0 +1,50 @@
...
+
+**Accounts Receivable Credit Account:** A control account of type receivable.
+
+**Accounts Receivable Discounted Account:** A receivable account for invoices which have been discounted.
+
+**Accounts Receivable Unpaid Account:** A receivable account for invoices which were discounted and remain unpaid even after the loan period is over.
+
+
+
+### 1. How to Post an Invoice Discounting Transaction
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
## 1. How to Post an Invoice Discounting Transaction
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md (hunk 12 above)
Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]: pending

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 23]
```diff
@@ -0,0 +1,50 @@
...
+
+1. Go to **Accounting > Banking and Payments > Invoice Discounting > New**.
+1. Enter Posting Date and Loan Start Date. Enter the Loan Period in days.
+1. Select invoices either manually in the table or by clicking on the 'Get Invoices' button on the top right.
+1. Select Short Term Loan Account, Bank Account, and Bank Charges Account.
+1. Select Accounts Receivable Credit Account, Accounts Receivable Discounted Account and Accounts Receivable Unpaid Account
+1. Click on Save then Submit.
+1. Click on 'Disburse Loan' and a Journal Entry screen will be prepared. Save then Submit the Journal Entry.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]: unclear
After submitting, click on 'Disburse Loan'. Go to Journal Entry....?

also, a screenshot for the journal entry screen?

**Comment by scmmishra** [comment]
@prasadarr and @asbasawaraj can you both collaborate over a call and get this merged?

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 24]
```diff
@@ -0,0 +1,50 @@
+<!-- add-breadcrumbs -->
+
+**Invoice discounting is the practice of using a company's unpaid invoices as collateral for a short term loan, which is issued by a bank or a finance company.**
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
**Invoice discounting is the practice of using a company's unpaid sales invoices as collateral for a short term loan, which is issued by a bank or a finance company.**
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md (hunk 23 above)
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion
1. After submitting, click on 'Disburse Loan'. You'll be taken to a Journal Entry screen will be prepared. Save then Submit the Journal Entry.
```

File: erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md [hunk 25]
```diff
@@ -0,0 +1,51 @@
+<!-- add-breadcrumbs -->
+
+**Invoice discounting is the practice of using a company's unpaid sales invoices as collateral for a short term loan, which is issued by a bank or a finance company.**
+To access the Invoice discounting list, go to:
+> Home > Accounting > Banking and Payments > Invoice Discounting
+### Prerequisites
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md]:
```suggestion

### Prerequisites
```

**Pull request merged**